addons/sale/models/sale.py

```diff
@@ -507,6 +507,7 @@ def _prepare_invoice(self):
             'team_id': self.team_id.id,
             'partner_id': self.partner_invoice_id.id,
             'partner_shipping_id': self.partner_shipping_id.id,
+            'invoice_partner_bank_id': self.company_id.partner_id.bank_ids[:1],
```

Comment:
To be consistent with the other Many2one:

```suggestion
            'invoice_partner_bank_id': self.company_id.partner_id.bank_ids[:1].id,
```

Reply:
updated, thanks for the review

Reply:
hum still not